chatgpt.el

```diff
@@ -110,6 +110,9 @@
 (defvar chatgpt-instances (ht-create)
   "List of instances, each pair is consist of (index . buffer).")
 
+(defvar-local chatgpt--window-prompt "Type response here..."
+  "Prompt shown to the user in the chat input window buffer.")
```

Comment:
It's better to use `defcustom` instead, and rename the variable to `chatgpt-window-prompt` (public).

Reply:
Got it -- I didn't care to make it public and just wanted a variable to avoid typing a literal string in more than one place.  All good though :)